commit 705951df7a609fe2fc5f7155dd68fc1a654e0868
Author: P33M <p33m@github.com>
Date:   Wed Jan 16 10:17:52 2019 +0000

    dwc_otg: fix bug with port_addr assignment for single-TT hubs
    
    See https://github.com/raspberrypi/linux/issues/2734
    
    The "Hub Port" field in the split transaction packet was always set
    to 1 for single-TT hubs. The majority of single-TT hub products
    apparently ignore this field and broadcast to all downstream enabled
    ports, which masked the issue. A subset of hub devices apparently
    need the port number to be exact or split transactions will fail.

diff --git a/drivers/usb/host/dwc_otg/dwc_otg_hcd_linux.c b/drivers/usb/host/dwc_otg/dwc_otg_hcd_linux.c
index c86a19ef8da4..23a9291b2614 100644
--- a/drivers/usb/host/dwc_otg/dwc_otg_hcd_linux.c
+++ b/drivers/usb/host/dwc_otg/dwc_otg_hcd_linux.c
@@ -232,7 +232,7 @@ static int _hub_info(dwc_otg_hcd_t * hcd, void *urb_handle, uint32_t * hub_addr,
 		else
 			*hub_addr = urb->dev->tt->hub->devnum;
 	}
-	*port_addr = urb->dev->tt->multi ? urb->dev->ttport : 1;
+	*port_addr = urb->dev->ttport;
    } else {
         *hub_addr = 0;
 	*port_addr = urb->dev->ttport;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1456_705951df7a609fe2fc5f7155dd68fc1a654e0868.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1456_705951df7a609fe2fc5f7155dd68fc1a654e0868.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
